mlflow/tracking/client.py

```diff
@@ -1732,6 +1732,10 @@ def log_artifact(self, run_id, local_path, artifact_path=None) -> None:
             is_dir: False
 
         """
+        if run_id.startswith("tr-"):
```

Comment:
Can we use `TRACE_REQUEST_ID_PREFIX`?

Reply:
good catch, didn't know we have this defined.